addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/requestor-max.yaml

```diff
@@ -6,4 +6,6 @@
         name:                          # String: Optional name for the request, for documentation only
         method:                        # String: A non-empty request method, default: GET
         data:                          # String: Optional data to send in the request body, supports vars
-        responseCode:                  # Int: An optional, expected response code against which the actual response code will be matched
\ No newline at end of file
+        responseCode:                  # Int: An optional, expected response code against which the actual response code will be matched
+        headers:                       # An optional list of headers
+            # - "header1:value1"
```

Comment:
Wouldn’t it make more sense for this to be between method and data?

Reply:
Sure, making the changes.

Reply:
I’m not sure how/where the plan generation is but that may also need to be re-ordered

Reply:
Ok